diff --git a/src/lib/fcitx/globalconfig.cpp b/src/lib/fcitx/globalconfig.cpp
index 49946c51..8c4c0e64 100644
--- a/src/lib/fcitx/globalconfig.cpp
+++ b/src/lib/fcitx/globalconfig.cpp
@@ -50,14 +50,14 @@ FCITX_CONFIGURATION(
         this,
         "EnumerateForwardKeys",
         _("Enumerate Input Method Forward"),
-        {},
+        {Key("Control+Shift_L")},
         KeyListConstrain({KeyConstrainFlag::AllowModifierLess,
                           KeyConstrainFlag::AllowModifierOnly})};
     KeyListOption enumerateBackwardKeys{
         this,
         "EnumerateBackwardKeys",
         _("Enumerate Input Method Backward"),
-        {},
+        {Key("Control+Shift_R")},
         KeyListConstrain({KeyConstrainFlag::AllowModifierLess,
                           KeyConstrainFlag::AllowModifierOnly})};
     Option<bool> enumerateSkipFirst{
