Index: fc/data/fcitx5-configtool.sh
===================================================================
--- fc.orig/data/fcitx5-configtool.sh
+++ fc/data/fcitx5-configtool.sh
@@ -135,7 +135,7 @@ run_dde() {
     local dbus_interface="org.deepin.dde.ControlCenter1"
     local dcc_module="keyboard/Manage Input Methods"

-    if dbus-send --print-reply=literal --dest=$dbus_service $dbus_path $dbus_interface.GetAllModule 2>> /dev/null | jq --arg url "$dcc_module" -Rse 'fromjson? // error("Bad input") | any(.url == $url)' >> /dev/null 2>&1; then
+    if dbus-send --print-reply=literal --dest=$dbus_service $dbus_path $dbus_interface.GetAllModule 2>> /dev/null | grep -q "$dcc_module" >> /dev/null 2>&1; then
         exec dbus-send --print-reply=literal --dest=$dbus_service $dbus_path $dbus_interface.ShowPage string:"$dcc_module"
     fi
     return 1
