Index: fcitx5/src/lib/fcitx/globalconfig.cpp
===================================================================
--- fcitx5.orig/src/lib/fcitx/globalconfig.cpp
+++ fcitx5/src/lib/fcitx/globalconfig.cpp
@@ -43,7 +43,7 @@ FCITX_CONFIGURATION(
         this,
         "AltTriggerKeys",
         _("Temporally switch between first and current Input Method"),
-        {Key("Shift_L")},
+        {},
         KeyListConstrain({KeyConstrainFlag::AllowModifierLess,
                           KeyConstrainFlag::AllowModifierOnly})};
     KeyListOption enumerateForwardKeys{
