Index: fcitx5-community/src/modules/notificationitem/dbusmenu.cpp
===================================================================
--- fcitx5-community.orig/src/modules/notificationitem/dbusmenu.cpp
+++ fcitx5-community/src/modules/notificationitem/dbusmenu.cpp
@@ -48,9 +48,9 @@ DBusMenu::~DBusMenu() = default;
 
 void DBusMenu::event(int32_t id, const std::string &type, const dbus::Variant &,
                      uint32_t) {
-    if (id == 0 && type == "opened") {
-        sendEventToTopLevel_ = true;
-    }
+    // if (id == 0 && type == "opened") {
+    //     sendEventToTopLevel_ = true;
+    // }
     // If top level menu is closed, reset the ic info.
     if (id == 0 && type == "closed") {
         lastRelevantIc_.unwatch();
